google/cloud/spanner/internal/time.h

```diff
@@ -42,26 +42,24 @@ std::chrono::nanoseconds FromProto(google::protobuf::Duration const& proto);
 /**
  * Convert a system_clock::time_point to a google::protobuf::Timestamp.
  */
-google::protobuf::Timestamp ToProto(std::chrono::system_clock::time_point tp);
+google::protobuf::Timestamp ToProto(Timestamp ts);
 
 /**
  * Convert a google::protobuf::Timestamp to a system_clock::time_point.
  */
-std::chrono::system_clock::time_point FromProto(
-    google::protobuf::Timestamp const& proto);
+Timestamp FromProto(google::protobuf::Timestamp const& proto);
 
 /**
  * Convert a system_clock::time_point to an RFC3339 "date-time".
  */
-std::string TimestampToString(std::chrono::system_clock::time_point tp);
+std::string TimestampToString(Timestamp ts);
 
 /**
  * Convert an RFC3339 "date-time" to a system_clock::time_point.
```

Comment:
nit: the comment does not match the function.

Reply:
Fixed.